### ISSUE TITLE
Support aggregating configuration files in one directory

### ISSUE THREAD

**Issue by suzuki-shunsuke** [aquaproj/aqua#1595]
## Feature Overview

Support aggregating configuration files in one directory.

e.g.

```
aquaproj/
  aqua.yaml
  aqua-checksums.json
  policy.yaml
  imports/
    terraform.yaml
    ...
```

Currently, aqua finds configuration file `^\.?aqua\.ya?ml$` from the current directory to the root directory.

https://aquaproj.github.io/docs/reference/config#configuration-file-path

I propose to add a directory to the target of configuration file paths.

## Why is the feature needed?

> Please explain the problem you want to solve.

aqua supports several configuration files.

```
aqua.yaml
aqua-checksums.json
aqua-policy.yaml
aqua/
  terraform.yaml
  ...
```

To be honest, I don't have any complaint about this, but I also understand some people want to aggregate configuration files in one directory. They don't want to add many files in the working directory.

## Directory file name

There are some candidates.

* `\.?aqua`: simplest, but I have a concern that the word `aqua` is too common.
* `\.?aquaproj`
* `\.?aquaproj-aqua`: a bit long

## Does the feature include Breaking Changes?

Basically no.

## Example Code

> command and configuration

```console
$ 
```

```yaml

```

**Comment by sheldonhull** [comment]
For consistency in other supporting tool .aqua-proj or such works for me.

**Comment by suzuki-shunsuke** [comment]
- https://github.com/aquaproj/aqua/pull/1615

**Comment by suzuki-shunsuke** [comment]
I prefer `aquaproj` than `aqua-proj` because

* `aquaproj` is this GitHub Organization name
* `aquaproj` looks cooler than `aqua-proj` for me

**Comment by suzuki-shunsuke** [comment]
- https://github.com/aquaproj/aqua/pull/1615

The following files are supported.

- aqua.yaml
- aqua.yml
- .aqua.yaml
- .aqua.yml
- aquaproj/aqua.yaml
- aquaproj/aqua.yml
- .aquaproj/aqua.yaml
- .aquaproj/aqua.yml

## Action List to aggregate configuration files in one directory

- Update aqua to v1.33.0 or later
- Update [update-checksum-action](https://github.com/aquaproj/update-checksum-action) to v0.1.4 or later
- Move files

```sh
mkdir aquaproj
cp -R aqua aquaproj/imports
mv aqua.yaml aqua-checksums.json aquaproj
rm -R aqua
vi aquaproj/aqua.yaml # Fix `import` if you change the directory name
cp aqua-policy.yaml aquaproj/policy.yaml # `cp` instead of `mv` to prevent an error
```

- Fix `AQUA_POLICY_CONFIG`
- After fixing `AQUA_POLICY_CONFIG`, remove the old policy file

```sh
rm aqua-policy.yaml
```

**Comment by suzuki-shunsuke** [comment]
- https://github.com/aquaproj/registry-tool/pull/282
- https://github.com/suzuki-shunsuke/github-comment/pull/780

**Comment by suzuki-shunsuke** [comment]
Hmm. I'm reconsidering the directory name.
Maybe `aqua` is better than `aquaproj`, because `aqua` is simple and easy to understand.
I had a concern the conflict with other projects, but the conflict would rarely occur.

**Comment by suzuki-shunsuke** [comment]
- https://github.com/aquaproj/aqua/pull/1626

**Comment by suzuki-shunsuke** [comment]
https://github.com/aquaproj/aqua/releases/tag/v1.33.0-3

**Comment by suzuki-shunsuke** [comment]
Released. https://github.com/aquaproj/aqua/releases/tag/v1.33.0
